core/lib/spree/core.rb

```diff
@@ -85,6 +85,15 @@ class GatewayError < RuntimeError; end
   end
 end
 
+if Gem::Version.new(Rails.version) < Gem::Version.new('5.2')
```

Comment:
`<=`?

Reply:
Oh, fixed it! :sweat: Thanks for spotting it! I've gone with `< 6.0` to catch patch-level versions.

Reply:
Smart, I actually almost suggested that. 😅